Sources/SmithyIdentity/AWSCredentialIdentityResolvers/CustomAWSCredentialIdentityResolver.swift

```diff
@@ -0,0 +1,25 @@
+//
```

Comment:
This was moved from aws-sdk-swift to here. It was moved bc this is not AWS specific, as in there's no underlying AWS service or AWS config for getting the credentials.

Reply:
AWS credentials are specifically called out by SRA as a smithy component, so this is good